signac/__main__.py

```diff
@@ -537,17 +537,25 @@ def _sig(st):
                 print(MSG_SYNC_STATS.format(stats=stats))
     except SchemaSyncConflict as error:
         _print_err(
-            "WARNING: The detected schemas of the two projects differ! "
-            "Use --force to ignore.")
-        only_in_src = error.schema_src.difference(error.schema_dst)
+            "WARNING: Merging two projects with different schema requires the -m/--merge option. "
```

Comment:
```suggestion
            "Synchronizing two projects with different schema requires the -m/--merge option. ")
```
This is not a warning, just a message.